library/modules/bigip_device_auth_radius.py

```diff
@@ -0,0 +1,765 @@
+    state: present
+    provider:
+      password: secret
+      server: lb.mydomain.com
+      user: admin
+  delegate_to: localhost
+'''
+
+RETURN = r'''
+servers:
+  description: The servers value of the resource.
+  returned: changed
+  type: complex
+  contains:
+    server:
+      description: ip address of RADIUS Server
+      type: str
+      sample: 1.1.1.1
+    port:
+      description: RADIUS service port
+      type: int
+      sample: 1812
+    timeout:
+      description: timeout value
+      type: int
+      sample: 3
+  sample: hash/dictionary of values
+service_type:
+  description: type of service requested from the RADIUS server.
+  returned: changed
+  type: str
+  sample: login
+retries:
+  description: number of authentication retries before authentication fails
+  type: int
+  sample: 10
+accounting_bug:
+  description: Enables or disables validation of the accounting response vector
+  type: bool
+  returned: changed
+  sample: enabled
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import env_fallback
+
+try:
+    from library.module_utils.network.f5.bigip import F5RestClient
+    from library.module_utils.network.f5.common import F5ModuleError
+    from library.module_utils.network.f5.common import AnsibleF5Parameters
+    from library.module_utils.network.f5.common import fq_name
+    from library.module_utils.network.f5.common import transform_name
+    from library.module_utils.network.f5.common import f5_argument_spec
+except ImportError:
+    from ansible_collections.f5networks.f5_modules.plugins.module_utils.bigip import F5RestClient
+    from ansible_collections.f5networks.f5_modules.plugins.module_utils.common import F5ModuleError
+    from ansible_collections.f5networks.f5_modules.plugins.module_utils.common import AnsibleF5Parameters
+    from ansible_collections.f5networks.f5_modules.plugins.module_utils.common import fq_name
+    from ansible_collections.f5networks.f5_modules.plugins.module_utils.common import transform_name
+    from ansible_collections.f5networks.f5_modules.plugins.module_utils.common import f5_argument_spec
+
+
+class Parameters(AnsibleF5Parameters):
+    api_map = {
+        'serviceType': 'service_type',
+        'accountingBug': 'accounting_bug'
+    }
+
+    api_attributes = [
+        'accountingBug',
+        'serviceType',
+        'servers',
+        'retries'
+    ]
+
+    returnables = [
+        'accounting_bug',
+        'servers',
+        'service_type',
+        'retries'
+    ]
+
+    updatables = [
+        'accounting_bug',
+        'servers',
+        'service_type',
+        'retries'
+    ]
+
+
+class ApiParameters(Parameters):
+    pass
+
+
+class ModuleParameters(Parameters):
+    @property
+    def accounting_bug(self):
+        if self._values['accounting_bug'] is None:
+            return 'disabled'
+        if self._values['accounting_bug']:
+            return 'enabled'
+        else:
+            return 'disabled'
```

Comment:
why do we return `disabled` when it has not been set? wouldn't this break idempotency? also look at how the booleans are treated in other modules for example:

https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_apm_acl.py#L457

Reply:
You are right! else statement will break idempotency in a case when account_bug is enabled. 

This comment is addressed. Thanks!